### ISSUE TITLE
Upper case on project names

### ISSUE THREAD

**Issue by MrMooky** [bdkjones/CodeKit#632]
This might not be labelled as a bug, but the latest update auto capitalises the first letter (it did this right after the update for every project in my list) even though the project names did not change. Because of the fact that I have multiple projects starting with lower case, this kind of triggers my inner monk. ._.

<img width="1122" alt="Screenshot 2020-10-27 at 16 42 36" src="https://user-images.githubusercontent.com/3603346/97325954-ca57f600-1873-11eb-9800-1858be9f1a0c.png">

**Comment by bdkjones** [comment]
Ha, yes, this is an intentional change. With the more compact layout of the projects popover and the increased spacing on Big Sur, names with lowercase leading letters looked misaligned next to the icon in their row. So CodeKit now forces the first letter to a capital. It doesn’t change anything on disk, however.

-Bryan

**Comment by MrMooky** [comment]
Mh, I see. I had hoped this was by mistake. Any chance you could implement a setting to disable that? It's basically what you see when opening CodeKit and to be honest, it really bugs me. I actually have two companies and multiple projects that have no uppercase letters in their name and having the first letter uppercase is just wrong with those names.

**Comment by greystate** [comment]
@MrMooky I know it's not a "solution" per se, but I also have a couple of clients whose names are specifically lowercase-initial-letter. What I ended up doing was to change their display names in the project settings to start with a _Zero-Width-Space_ - which I haven't yet had any issues with.

Sorry, I know it's a terrible hack 🥇

**Comment by bdkjones** [comment]
Ha, that's actually really clever.

**Comment by MrMooky** [comment]
@greystate It basically works, but as soon as I "initialise" the project, thus refresh, the white space seems to be removed by CodeKit again.

**Comment by greystate** [comment]
@MrMooky That's weird — I did try to see if I could make it go away, but I can't... I've tried restarting the app etc. — it works fine for me?

Are you removing the project from CodeKit and adding it back in? (Even though that shouldn't change anything - the character is in the project file...)

**Comment by MrMooky** [comment]
@greystate Ah, my bad. I changed the name when the project was not activated, so the config file was not updated and the change was only visible in the current state. So when I activated the project, the "old" config file was used without the space character. So yeah, really terrible solution but it works for now. 😄 👍 

Would still be cool if this was not mandatory and just like the way it was before.

> With the more compact layout of the projects popover and the increased spacing on Big Sur, names with lowercase leading letters looked misaligned next to the icon in their row.

I think it looks just fine.

<img width="438" alt="Screenshot 2020-11-20 at 13 36 20" src="https://user-images.githubusercontent.com/3603346/99800924-73250880-2b35-11eb-84de-4fb66ef80386.png">
